### ISSUE TITLE
GOG and Steam Auto-Detection?

### ISSUE THREAD

**Issue by MasterO2** [LTCHIPS/rottexpr#23]
If you install a game on your computer via a GOG.com installer, sourceports for that game can often autodetect the install without you having to do anything else.

For example, if I install Doom 2's GOG edition with the GOG installer and I then run GZdoom, GZdoom will then detect that the GOG edition is installed on the system.

Would something like this be possible for Rottexpr with the GOG and Steam editions of Rise of the Triad: Dark War?

**Comment by LTCHIPS** [comment]
Yes, it would be possible. I haven't made any plans to implement something like this yet though, but could be a good idea for speeding up/simplifying the setup process for rottexpr.
